### PR TITLE
Remove download from loop

### PR DESCRIPTION
**Pull request by AWare**
Calling toDataURL was eating performance with big images.

### CODE REVIEW

File: src/canvas.js [hunk 1]
```diff
@@ -111,13 +111,14 @@ class CanvasCard {
...
-    lineHeight
+    lineHeight,
+    scale
   }) {
     canvasContext.font = `${fontSize}px ${font}`;
 
     lines.forEach((line, i) => {
       const yOffset = initialOffset + lineHeight * (i + 1);
-      canvasContext.fillText(line, Config.padding, yOffset);
+      canvasContext.fillText(line, Config.padding * scale, yOffset);
```
Comment (akash1810) [inline src/canvas.js]: Could we split this into its own PR? That is, I'm not sure how this relates to changing the way we're downloading?

Comment (AWare) [reply src/canvas.js]: done

**Comment by akash1810** [comment]
What @prout-bot ?! This *IS* on PROD... 🤔
